_layouts/tutorial.html

```diff
@@ -83,6 +113,36 @@ <h3 class="title">Code example:</h3>
             Basic Kafka
           </a>
           {% endif %}
+          {% if site.data.tutorials[page.static_data].status.scala == 'enabled' %}
+          <a class="button {% if page.stack == 'scala' %} is-selected {% endif %}" href="{{ site.data.tutorials[page.static_data].slug | relative_url }}/scala.html">
+            Scala
+          </a>
```

Comment:
Maybe just add the `scala` section and we'll add the other language blocks when we get tutorials.  I don't have a strong opinion on this though.